_posts/fork/2021-12-21-Deepash.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: theme
+title: DeepAsh
+category: fork
+homepage: https://github.com/LJjia/typora.theme.DeepAsh
+download: https://github.com/LJjia/typora.theme.DeepAsh/releases/latest
+built-in: false
+author: LJjia
+thumbnail: deepash.png
+typora-root-url: ../../
+typora-copy-images-to: ../../media/theme/deepash
+---
+
+# Deepash
+
+Base on official Ash theme,but add some vibrant colors.
```

Comment:
In that case, please add an link on the ash theme

Reply:
ok,I add a new [pull request](https://github.com/typora/typora-theme-gallery/pull/253).
By the way,can I get a free license after submitting the theme?
How do I get it?